plugins/block-sharable-procedures/src/index.ts

```diff
@@ -9,6 +9,15 @@ import {blocks} from './blocks';
 import {IProcedureBlock, isProcedureBlock} from './i_procedure_block';
 import {ObservableParameterModel} from './observable_parameter_model';
 import {ObservableProcedureModel} from './observable_procedure_model';
+import {ProcedureBase, ProcedureBaseJson} from './events_procedure_base';
+import {ProcedureChangeReturn, ProcedureChangeReturnJson} from './events_procedure_change_return';
+import {ProcedureCreate, ProcedureCreateJson} from './events_procedure_create';
+import {ProcedureDelete, ProcedureDeleteJson} from './events_procedure_delete';
+import {ProcedureParameterBase, ProcedureParameterBaseJson} from './events_procedure_parameter_base';
+import {ProcedureParameterCreate, ProcedureParameterCreateJson} from './events_procedure_parameter_create';
+import {ProcedureParameterDelete, ProcedureParameterDeleteJson} from './events_procedure_parameter_delete';
+import {ProcedureParameterRename, ProcedureParameterRenameJson} from './events_procedure_parameter_rename';
+import {ProcedureRename, ProcedureRenameJson} from './events_procedure_rename';
```

Comment:
Any particular reason you have opted to do the reexports this way, with a bunch of `import`s followed by a single `export`?  I'm aware that we have some usage of this style in core, but I think it is mostly a hold-over from when those files were closure modules.  The styleguide doesn't seem give any definitive rules AFACT but I note that [it has one example of `export {…} from '…'`](https://google.github.io/styleguide/tsguide.html#exports) and none of `export {…}`.

```suggestion
export {ProcedureBase, ProcedureBaseJson} from './events_procedure_base';
export {ProcedureChangeReturn, ProcedureChangeReturnJson} from './events_procedure_change_return';
export {ProcedureCreate, ProcedureCreateJson} from './events_procedure_create';
export {ProcedureDelete, ProcedureDeleteJson} from './events_procedure_delete';
export {ProcedureParameterBase, ProcedureParameterBaseJson} from './events_procedure_parameter_base';
export {ProcedureParameterCreate, ProcedureParameterCreateJson} from './events_procedure_parameter_create';
export {ProcedureParameterDelete, ProcedureParameterDeleteJson} from './events_procedure_parameter_delete';
export {ProcedureParameterRename, ProcedureParameterRenameJson} from './events_procedure_parameter_rename';
export {ProcedureRename, ProcedureRenameJson} from './events_procedure_rename';
```

Reply:
Fixed!